### DISCUSSION TITLE
Migration to Magento from Shopify

### DISCUSSION THREAD

**Discussion by zhang-min-dev** [community/community#103009] [Programming Help] [answered]
### Body

Hello everyone!

How to import/export the product csv file from Magento to Shopify?
I just imported Magento csv file into Shopify, so all product images are broken.
How should I do?
Please help me.

Thank you

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by liquidshopier** [comment, marked as answer]
Hello Zhang,

If you are migrating the product csv file from Magento to Shopify, you should import using Matrixify app.
also you can see this tutorial: https://matrixify.app/tutorials/migrate-products-from-magento-to-shopify/
I hope you find guide helpful.

**Reply by zhang-min-dev** [reply]
Hi liquidshopier,

Thank you.
